rules/restoration/src/Type/ConstantReturnToParamTypeConverter.php

```diff
@@ -44,11 +44,14 @@ private function unwrapConstantTypeToObjectType(Type $type): Type
         if ($type instanceof UnionType) {
             $types = [];
             foreach ($type->getTypes() as $unionedType) {
-                $types[] = $this->unwrapConstantTypeToObjectType($unionedType);
+                $type = $this->unwrapConstantTypeToObjectType($unionedType);
+                if ($type !== null) {
+                    $types[] = $type;
+                }
             }
             return $this->typeFactory->createMixedPassedOrUnionType($types);
         }
 
-        throw new NotImplementedYetException();
+        return null;
```

Comment:
For unknown type we use PHPStan convetions of `MixedType`.
Can we apply it here?

Reply:
I haven't tried that, but my guess is that that will modify the `@param` annotation to contain `mixed`.

Reply:
Okay, I just tried that and my suspicion was correct. So I think `null` is better, meaning, there is actually no type to be derived.